### PR TITLE
DM-1013 Fix properties in OPC UA failure handling

### PR DESCRIPTION
**Pull request by sag-eweingaertner**
Another OPC UA doc fix.

### CODE REVIEW

File: content/protocol-integration/opcua-bundle/device-protocols.md [hunk 1]
```diff
@@ -179,26 +179,18 @@ When a device protocol has been applied to or un-applied from a node, a monitori
...
-
-- maxRetries[number] - Number of retries for failed queues; if the maximum is reached the queue will be saved as permanently failed and never retried again, default is 5.
-
-- pendingMaxAge[seconds] - Queues with a timestamp older than this will not be retried regardless of the retry status, default is 86400 .
+|<div style="width:200px">Setting</div>|Value format|Example |Description|																				
+|-----------|------------|--------|-----------|
+|enabled  | boolean (true/false) | true | Activate or deactivate the fail over for custom actions |
+|maxRetries| number | 5 | Number of retries for failed queues; if the maximum is reached the queue will be saved as permanently failed and never retried again, default is 5.|
+|noRetryHttpCodes| Comma-separated list of HTTP response codes | 400,500 |  If retries are enabled (failureHandling.enabled=true), this setting allows retries to be skipped for certain HTTP response codes. In the given example invocations to the HTTP endpoint being answered with `400 Bad Request` or `500 Internal server error` would not be reattempted.|
```
Comment (rssag) [inline content/protocol-integration/opcua-bundle/device-protocols.md:195]:
```suggestion
|noRetryHttpCodes| Comma-separated list of HTTP response codes | 400,500 |  If retries are enabled (failureHandling.enabled=true), this setting allows retries to be skipped for certain HTTP response codes. In the given example invocations to the HTTP endpoint that receive a `400 Bad Request` or `500 Internal server error` response are skipped.|
```

File: content/protocol-integration/opcua-bundle/device-protocols.md [hunk 2]
```diff
@@ -179,26 +179,18 @@ When a device protocol has been applied to or un-applied from a node, a monitori
...
-
-- failedCustomActionQueueSize[number] - Size of maximum elements in one queue; if the limit is reached the queue will be saved, default is 100.
-
-- maxRetries[number] - Number of retries for failed queues; if the maximum is reached the queue will be saved as permanently failed and never retried again, default is 5.
-
-- pendingMaxAge[seconds] - Queues with a timestamp older than this will not be retried regardless of the retry status, default is 86400 .
+|<div style="width:200px">Setting</div>|Value format|Example |Description|																				
+|-----------|------------|--------|-----------|
+|enabled  | boolean (true/false) | true | Activate or deactivate the fail over for custom actions |
+|maxRetries| number | 5 | Number of retries for failed queues; if the maximum is reached the queue will be saved as permanently failed and never retried again, default is 5.|
```
Comment (rssag) [inline content/protocol-integration/opcua-bundle/device-protocols.md:194]:
```suggestion
|maxRetries| number | 5 | Number of retries for failed queues. If the maximum is reached the queue is saved as permanently failed and never retried again. Default is 5.|
```

File: content/protocol-integration/opcua-bundle/device-protocols.md [hunk 3]
```diff
@@ -179,26 +179,18 @@ When a device protocol has been applied to or un-applied from a node, a monitori
...
 
-Queues are collections of failed custom actions, including the complete HTTP request of this custom action. Each entry of the queue is one failed custom action. The collection has a defined size in _failedCustomActionQueueSize_ and a maximum number of retries in _maxRetries_.  
+Queues are collections of failed custom actions, including the complete HTTP request of this custom action. Each entry of the queue is one failed custom action. 
 
 A background scheduler task will retry each queue up to the number of _maxRetries_. If _maxRetries_ is reached the queue will be stored as a permanently failed queue in the event repository.
 
-All elements of the queue will be retried, so the count of the elements in the queue will be decreasing with each successful retried custom action. These queues are also timing out when the reach they _pendingMaxAge_ to reduce the load of the scheduler task.
+All elements of the queue will be retried after the retry delay has passed. In effect, the count of the elements in the queue will be decreasing with each successful retried custom action.
```
Comment (rssag) [inline content/protocol-integration/opcua-bundle/device-protocols.md:186]:
```suggestion
All elements of the queue are retried after the retry delay has passed. In effect, the count of the elements in the queue is decreases with each successful retried custom action. 
```

**Review by rssag — CHANGES_REQUESTED** [review]

**Comment by rssag** [comment]
what are the target versions? @sag-eweingaertner

**Comment by sag-eweingaertner** [comment]
> what are the target versions? @sag-eweingaertner

Added 10.17 and auto-backport.

**Review by rssag — APPROVED** [review]

**Review by bggov — APPROVED** [review]

File: content/protocol-integration/opcua-bundle/device-protocols.md [hunk 4]
```diff
@@ -179,26 +179,18 @@ When a device protocol has been applied to or un-applied from a node, a monitori
...
+Queues are collections of failed custom actions, including the complete HTTP request of this custom action. Each entry of the queue is one failed custom action. 
 
 A background scheduler task will retry each queue up to the number of _maxRetries_. If _maxRetries_ is reached the queue will be stored as a permanently failed queue in the event repository.
 
-All elements of the queue will be retried, so the count of the elements in the queue will be decreasing with each successful retried custom action. These queues are also timing out when the reach they _pendingMaxAge_ to reduce the load of the scheduler task.
+All elements of the queue are retried after the retry delay has passed. In effect, the count of the elements in the queue is decreases with each successful retried custom action. 
 
-The following parameters can be set:  
+In the gateway configuration (see also [Gateway Configuration: Additional customization](#additional-customizations)) the following parameters can be configured:
```
Comment (Oezge-Akin) [inline content/protocol-integration/opcua-bundle/device-protocols.md:188]: property group information is not mentioned.
gateway.mappingExecution.http.failureHandling

File: content/protocol-integration/opcua-bundle/device-protocols.md [hunk 5]
```diff
@@ -179,26 +179,18 @@ When a device protocol has been applied to or un-applied from a node, a monitori
...
-- reScheduleElements[number] - Number of queues loaded for retry at the same time, default is 100.
-
-- failedCustomActionQueueSize[number] - Size of maximum elements in one queue; if the limit is reached the queue will be saved, default is 100.
-
-- maxRetries[number] - Number of retries for failed queues; if the maximum is reached the queue will be saved as permanently failed and never retried again, default is 5.
-
-- pendingMaxAge[seconds] - Queues with a timestamp older than this will not be retried regardless of the retry status, default is 86400 .
+|<div style="width:200px">Setting</div>|Value format|Example |Description|																				
+|-----------|------------|--------|-----------|
+|enabled  | boolean (true/false) | true | Activate or deactivate the fail over for custom actions |
```
Comment (Oezge-Akin) [inline content/protocol-integration/opcua-bundle/device-protocols.md:193]: Information about the defaults are missing.

**Comment by rssag** [comment]
cherry-picked manually.

**Pull request merged**